### DISCUSSION TITLE
My account was deleted for no reason

### DISCUSSION THREAD

**Discussion by tanwait** [community/community#21861] [New to GitHub]
<p> In May 11, my account can still log in normally.I cannot log in normally from May 12 till now,I’m sure my account and password are correct.I clicked on the <em>forget password</em>, GitHub redirected to the reset password page, and click on <em>send password reset email</em>, but GitHub showed that my email was not found. Can someone tell me what happened and help me?Here is the screenshot.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/f034c2a20488d2f9fc72d39bec85e5cfae66c4ae.png" data-download-href="/uploads/short-url/ygXD8HBKUNedfnL4jVBOYJskELI.png?dl=1" title="微信截图_20180514112857.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26588586/181067341-ad818a9b-e8e7-490a-ad95-cf6b585fa23e.png" alt="微信截图_20180514112857.png" data-base62-sha1="ygXD8HBKUNedfnL4jVBOYJskELI" width="690" height="399" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/f034c2a20488d2f9fc72d39bec85e5cfae66c4ae_2_690x399.png, https://aws1.discourse-cdn.com/github/optimized/1X/f034c2a20488d2f9fc72d39bec85e5cfae66c4ae_2_1035x598.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/f034c2a20488d2f9fc72d39bec85e5cfae66c4ae_2_1380x798.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/f034c2a20488d2f9fc72d39bec85e5cfae66c4ae_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">微信截图_20180514112857.png</span><span class="informations">1437×831 20.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4641167be9ba92824ff8bde306915e11222d5fa5.png" data-download-href="/uploads/short-url/a1uS7ETqWFokXehyJCH34B86D7D.png?dl=1" title="微信截图_20180514113502.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26588586/181067343-8f513646-1776-4722-bf82-c3619b15e01f.png" alt="微信截图_20180514113502.png" data-base62-sha1="a1uS7ETqWFokXehyJCH34B86D7D" width="690" height="399" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/4641167be9ba92824ff8bde306915e11222d5fa5_2_690x399.png, https://aws1.discourse-cdn.com/github/optimized/1X/4641167be9ba92824ff8bde306915e11222d5fa5_2_1035x598.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/4641167be9ba92824ff8bde306915e11222d5fa5_2_1380x798.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4641167be9ba92824ff8bde306915e11222d5fa5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">微信截图_20180514113502.png</span><span class="informations">1439×833 21.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/94f6b3196aec614bb76707c989be4040ca6e6adf.png" data-download-href="/uploads/short-url/lfNdsw2qOwmM5GJFW6gD4cZf2iz.png?dl=1" title="微信截图_20180514113515.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/26588586/181067345-f0b16295-a7bb-4b30-b538-bc6be3315dfa.png" alt="微信截图_20180514113515.png" data-base62-sha1="lfNdsw2qOwmM5GJFW6gD4cZf2iz" width="690" height="398" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/94f6b3196aec614bb76707c989be4040ca6e6adf_2_690x398.png, https://aws1.discourse-cdn.com/github/optimized/1X/94f6b3196aec614bb76707c989be4040ca6e6adf_2_1035x597.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/94f6b3196aec614bb76707c989be4040ca6e6adf_2_1380x796.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/94f6b3196aec614bb76707c989be4040ca6e6adf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">微信截图_20180514113515.png</span><span class="informations">1439×832 23.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/tanwait">@tanwait</a>,</p>
<p>Please contact <a href="mailto:support@github.com">support@github.com</a> for this. The forum community members don’t have any access to systems that could help you.</p>
<p>Good luck!</p>

**Comment by tanwait** [comment]
<p><a class="mention" href="/seveas">@seveas</a> Thank you for your reply, I have contacted <a href="mailto:support@github.com">support@github.com</a> and they have responsed to me.</p>

**Reply by devkkaran** [reply]
[@seveas](https://github.com/seveas) I recently have same issue. my account is not showing anywhere. i generate ticket also and mail to this mail id [support@github.com](mailto:support@github.com) and also to this mail id: [support@githubsupport.com](mailtosupport@githubsupport.com) . [@tanwait](https://github.com/tanwait) Reply if you solve your problem and Get Back your Git Hub Account...

**Reply by peroskyX** [reply]
@devkkaran did have you been able to resolve the issues cause i also the same issues

**Reply by ali321213** [reply]
@tanwait Did your issue resolved?
